master/reference/felix/configuration.md

```diff
@@ -50,6 +50,7 @@ The full list of parameters which can be set is as follows.
 | `PrometheusProcessMetricsEnabled` | `FELIX_PROMETHEUSPROCESSMETRICSENABLED` | Set to `false` to disable process metrics collection, which the Prometheus client does by default. This reduces the number of metrics reported, reducing Prometheus load. [Default: `true`] | boolean |
 | `ReportingIntervalSecs`           | `FELIX_REPORTINGINTERVALSECS`           | Interval at which Felix reports its status into the datastore or `0` to disable. Must be non-zero in OpenStack deployments. [Default: `30`] | int |
 | `ReportingTTLSecs`                | `FELIX_REPORTINGTTLSECS`                | Time-to-live setting for process-wide status reports. [Default: `90`] | int |
+| `SockmapEnabled`                  | `FELIX_SOCKMAPENABLED`                  | Enable experimental Sockmap acceleration between application and proxy sidecar when using [application layer policy](https://github.com/projectcalico/app-policy). [Default: `false`] | boolean |
```

Comment:
Should probably use the same URL as below?

Reply:
I was following the convention on this file, but maybe it was using that link because the documentation wasn't ready back then, I'll update.